services/bettersocial/user/BetterSocialBlockUserV2.js

```diff
@@ -89,6 +89,7 @@ const BetterSocialBlockUserV2 = async (token, selfUserId, targetUserId, source,
   }
 
   BetterSocialScoreBlockUser(selfUserId, targetUserId, postId);
+  BetterSocialScoreBlockUser(selfUserId, targetAnonymousUserId?.user_id, '');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `BetterSocialScoreBlockUser` is being called with an empty string as the third argument. This could potentially lead to unexpected behavior if the function does not handle this case properly. It would be better to pass a more meaningful value or `null` if it's acceptable and handled in the function.
```diff
-   BetterSocialScoreBlockUser(selfUserId, targetAnonymousUserId?.user_id, '');
+   BetterSocialScoreBlockUser(selfUserId, targetAnonymousUserId?.user_id, null);
```
Also, there is no null check for `targetAnonymousUserId`. If `targetAnonymousUserId` is `null` or `undefined`, accessing `user_id` property will throw an error. Consider adding a null check before accessing its properties.
```diff
-   BetterSocialScoreBlockUser(selfUserId, targetAnonymousUserId?.user_id, null);
+   BetterSocialScoreBlockUser(selfUserId, targetAnonymousUserId ? targetAnonymousUserId.user_id : null, null);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->